com.unity.render-pipelines.universal/Runtime/2D/Passes/Render2DLightingPass.cs

```diff
@@ -176,6 +177,11 @@ private void GetTransparencySortingMode(Camera camera, ref SortingSettings sorti
                 }
             }
 
+            foreach (var layerBatch in layerBatches)
```

Comment:
No need to loop through all the instances here, right? Only instances from `startIndex` to `batchCount` were used.

Reply:
fixed